SingularityServiceIntegrationTests/pom.xml

```diff
@@ -185,7 +190,7 @@
                       <MESOS_MASTER>zk://singularity-test-zk:2181/mesos</MESOS_MASTER>
                       <MESOS_CONTAINERIZERS>docker,mesos</MESOS_CONTAINERIZERS>
                       <MESOS_HOSTNAME>${mesos.slave2.hostname}</MESOS_HOSTNAME>
-                      <MESOS_PORT>5052</MESOS_PORT>
+                      <MESOS_PORT>5051</MESOS_PORT>
```

Comment:
my mistake -- need to update that setting too. the reason why i'm changing ports is that I noticed that we have slave ports hardcoded in `MesosClient`: https://github.com/HubSpot/Singularity/blob/master/SingularityMesosClient/src/main/java/com/hubspot/mesos/client/MesosClient.java#L28

we can fix the hardcoded port in a separate PR